CONTRIBUTING.md

```diff
@@ -240,15 +240,16 @@ There are a few main venues of communication:
 * For more organizational concerns, the [mailing
   list](http://www.haskell.org/mailman/listinfo/cabal-devel) is used.
 
-* Many developers idle on `#hackage` on `irc.freenode.net` ([archives](http://ircbrowse.net/browse/hackage)).  `#ghc` ([archives](http://ircbrowse.net/browse/ghc)) is also a decently good bet.
+* Many developers idle on `#hackage` on [irc.libera.net](https://libera.chat) ([archives](http://ircbrowse.net/browse/hackage)).  `#ghc` ([archives](http://ircbrowse.net/browse/ghc)) is also a decently good bet.
```

Comment:
archives links are broken and i guess it should be updated to point to actual ones, if they even exists

Reply:
We could use matrix for that (if we do want archives) https://view.matrix.org/room/%21VeTpfwcnSnUZwjyccf:libera.chat/

Also s/libera.net/libera.chat/

Reply:
```suggestion
* Many developers idle on `#hackage` on https://libera.chat ([archives](http://ircbrowse.net/browse/hackage)).  `#ghc` ([archives](http://ircbrowse.net/browse/ghc)) is also a decently good bet.
```

Reply:
> We could use matrix for that (if we do want archives) https://view.matrix.org/room/%21VeTpfwcnSnUZwjyccf:libera.chat/

It gives to me:

```
User @2985302:matrix.org not in room !VeTpfwcnSnUZwjyccf:libera.chat, and room previews are disabled (M_FORBIDDEN)
```